doc/migration/version-upgrade-guide.md

```diff
@@ -973,6 +977,33 @@ With:
 - name: {{your operator name which is the value of metadata.name in this file}}
 ```
 
+### Breaking Changes on Commands
+
+This release contains breaking changes in some commands.
+
+- The --namespace flag from operator-sdk run --local and operator-sdk test --local command was deprecated and was replaced by --watch-namespace and --operator-namespace .
+
+Note that --watch-namespace can be used to set the namespace(s) which the operator will watch for changes. It will set the environment variable WATCH_NAMESPACE. Use explicitly an empty string to watch all namespaces or inform a List of namespaces such as "ns1,ns2" when the operator is cluster-scoped. If you use a List, then it needs contains the namespace where the operator is "deployed" in since the default metrics implementation will manage resources in the Operator's namespace. By default, it will be the Operator Namespace.
+
+Then, use the flag --operator-namespace to inform the namespace where the Operator will be "deployed" in and then, it will set the environment variable OPERATOR_NAMESPACE. If this value is not set, then it will be the namespace defined as default in the Kubeconfig.
+
+### Notable Changes
```

Comment:
Missing remove the `### Notable Changes`

Reply:
done